src/index.js

```diff
@@ -33,6 +32,7 @@ const {
   errorHandler,
   ipWhitelist,
   apimapFieldsFormater,
+  schemaFileUpdater,
 } = context.inject();
```

Comment:
`inject` must be followed by
`if(!schemaFileUpdater) throw new Error('missing schemaFileUpdater')`

In order to stop now in case of dependency missing, and pointing the location.

Reply:
Thank you! I was not aware of this rule (I guess "must" is a rule flag)
 - Does this mean it has to be done everywhere we have a `const { x, y, z } = inject()`?
 - I don't see an example of this pattern in this actual file (nor elsewhere in the repository). Does this mean `schemaFileUpdater` is special (vs `logger,pathService,errorHandler,ipWhitelist,apimapFieldsFormater` that are not tested) or is this a mistake?

Reply:
I think all the repo does not follow the rule :) 
Asserting dependencies is important, yes, to early and properly fail.